src/shared/inc/sospriv.idl

```diff
@@ -465,12 +494,28 @@ interface ISOSDacInterface12 : IUnknown
 [
     object,
     local,
-    uuid(3176a8ed-597b-4f54-a71f-83695c6a8c5d)
+    uuid(3176a8ed-597b-4f54-a71f-83695c6a8c5e)
 ]
 interface ISOSDacInterface13 : IUnknown
 {
     HRESULT TraverseLoaderHeap(CLRDATA_ADDRESS loaderHeapAddr, LoaderHeapKind kind, VISITHEAP pCallback);
     HRESULT GetDomainLoaderAllocator(CLRDATA_ADDRESS domainAddress, CLRDATA_ADDRESS *pLoaderAllocator);
     HRESULT GetLoaderAllocatorHeapNames(int count, const char **ppNames, int *pNeeded);
     HRESULT GetLoaderAllocatorHeaps(CLRDATA_ADDRESS loaderAllocator, int count, CLRDATA_ADDRESS *pLoaderHeaps, LoaderHeapKind *pKinds, int *pNeeded);
+    HRESULT GetHandleTableMemoryRegions(ISOSMemoryEnum **ppEnum);
+    HRESULT GetGCBookkeepingMemoryRegions(ISOSMemoryEnum **ppEnum);
+    HRESULT GetGCFreeRegions(ISOSMemoryEnum **ppEnum);
+    HRESULT LockedFlush();
+}
+
+[
+    object,
+    local,
+    uuid(9aa22aca-6dc6-4a0c-b4e0-70d2416b9837)
+]
+interface ISOSDacInterface14 : IUnknown
+{
+    HRESULT GetStaticBaseAddress(CLRDATA_ADDRESS methodTable, BOOL isGCStaticBase, CLRDATA_ADDRESS *address);
```

Comment:
@leculver, eventually CLRMD will need to be updated to use these new DAC APIs for .NET 9.0.